buildbot-buildmaster/traefik/dynamic/tls.yml

```diff
@@ -0,0 +1,23 @@
+# tls options. derived from toml-file generated by:
+# https://ssl-config.mozilla.org/#server=traefik&version=2&config=intermediate&guideline=5.4
+
+http:
+  middlewares:
+    hsts-header:
+      headers:
+        customResponseHeaders:
+          Strict-Transport-Security: "max-age=63072000"
+        stsPreload: true
+tls:
+  options:
+    default:
+      minVersion:  "VersionTLS12"
+      sniStrict: true
+      cipherSuites:
```

Comment:
This list is very short and specific. Could this be the reason for the TLS issues? Please check.